client-libraries.md

```diff
@@ -0,0 +1,28 @@
+# IPFS Client Libraries
+
+For a list of IPFS implementations, please refer to *[implementations](https://github.org/ipfs/ipfs/implementations.md).
+
+| Language | Client Library |
+|----------|----------------|
+| Go | https://github.com/ipfs/go-ipfs-api
+| Java | https://github.com/ipfs/java-ipfs-api |
+| Javascript | https://github.com/ipfs/js-ipfs-api |
+| Python | https://github.com/ipfs/python-ipfs-api |
+| Scala | https://github.com/ipfs/scala-ipfs-api |
+| Haskell | https://github.com/davidar/hs-ipfs-api |
+| Swift | https://github.com/ipfs/swift-ipfs-api |
+| CommonLisp | https://github.com/WeMeetAgain/cl-ipfs-api |
+| Rust | https://github.com/rschulman/rust-ipfs-api |
+| Ruby | https://github.com/Fryie/ipfs-ruby |
+| Swift | https://github.com/NeoTeo/ipfs-osx-service |
+| PHP | https://github.com/cloutier/php-ipfs-api |
+| C/C++ | contact: [@PayasR](https://github.com/PayasR) |
+| Julia | contact: [@rened](https://github.com/rened) |
+| Lua | contact: [@seclorum](https://github.com/seclorum) |
+| Erlang | ! |
+
+ObjC
+Swift
+Scala - https://github.com/ipfs/scala-ipfs-api
+
+Want to contribute one of the above client libraries? [Tell us](https://github.com/ipfs/ipfs/issues/83) so we can help!
```

Comment:
Worth including a link to:  https://github.com/ipfs/specs/blob/master/overviews/implement-api-bindings.md -- though nothing there yet, merging tomorrow: https://github.com/ipfs/specs/blob/overviews/overviews/implement-api-bindings.md for now